tests/app.js

```diff
@@ -29,6 +29,7 @@ import '@react-native-firebase/database';
 import '@react-native-firebase/dynamic-links';
 import '@react-native-firebase/firestore';
 import * as functionsModular from '@react-native-firebase/functions';
+import * as storageModular from '@react-native-firebase/storage';
```

Comment:
Really minor point - but can you snuggle this one up under the non-modular storage import just a few lines below? These imports have been re-positioned a couple times and right now we have it organized such that the same module is imported modular/non-modular as a unit?